packages/react/README.md

```diff
@@ -27,6 +28,9 @@ import torusModule from '@web3-onboard/torus'
 import keepkeyModule from '@web3-onboard/keepkey'
 import dcentModule from '@web3-onboard/dcent'
 
+// Sign up to get your free API key at https://www.blocknative.com/pricing
```

Comment:
@Adamj1232 not your fault but the words "free" and "/pricing" in the same sentence like this IMO is a huge turn off